runtime/doc/tags

```diff
@@ -5716,7 +5716,6 @@ H	motion.txt	/*H*
 Haiku	os_haiku.txt	/*Haiku*
 HelpToc-mappings	helphelp.txt	/*HelpToc-mappings*
 I	insert.txt	/*I*
-ICCF	uganda.txt	/*ICCF*
```

Comment:
I think it is probably a good idea to keep the old ICCF help tags so someone typing `:h iccf` will be redirected to the updated `kuwasha` help page.